extra/query_propagation.rst

```diff
@@ -80,14 +80,17 @@ A useful companion to :code:`run_command_on_placements` is :code:`run_command_on
   -- We want to synchronise them so that every time little_vals
   -- are created, big_vals appear with double the value
   --
-  -- First we make a trigger function for each placement
-  SELECT run_command_on_placements('big_vals', $cmd$
-    CREATE OR REPLACE FUNCTION embiggen_%1$I() RETURNS TRIGGER AS $$
+  -- First we make a trigger function on each worker, which will
+  -- take the destination table placement as an argument
+  SELECT run_command_on_workers($cmd$
+    CREATE OR REPLACE FUNCTION embiggen() RETURNS TRIGGER AS $$
```

Comment:
Trigger functions are not supposed to declare their arguments, or else the type won't match up. That's why we leave the args list empty and consult TG_ARGV.